sections/performance/nativeoverutil.md

```diff
@@ -0,0 +1,76 @@
+# Prefer native JS methods over user-land utils like Lodash
+
+
+<br/><br/>
+
+### One Paragraph Explainer
+
+Sometimes, using native methods is better than requiring `lodash` or `underscore` because it **will not lead in a performance boost** and use more space than necessary.
+The performance using native methods result in an overall ~50% gain which includes the following methods:
+ - `Array.concat`
+ - `Array.fill`
+ - `Array.filter`
+ - `Array.map`
+ - `(Array|String).indexOf`
+ - `Object.find`
+ - ...
+
+
+<!-- comp here: https://gist.github.com/Berkmann18/3a99f308d58535ab0719ac8fc3c3b8bb-->
+
+<br/><br/>
+
+### Example: benchmark comparison - Lodash vs V8 (Native)
+The graph below shows the mean of the benchmarks for a variety of Lodash methods, this shows that Lodash methods take on average 146.23% more time to complete the same tasks as V8 methods.
+
+![meanDiag](../../assets/images/sampleMeanDiag.png)
```

Comment:
Let's include a link to the article?

Reply:
I've added a link to the relevant spreadsheet if that's fine?